docs/topics/spiders.rst

```diff
@@ -145,6 +145,11 @@ scrapy.Spider
            <topics-addons>`. The final settings are available in the
            :meth:`start_requests` method and later.
 
+           It's important to note that the :param crawler: in the :meth:`scrapy.Spider.from_crawler`
+           does not have most of its components fully initialized.
+           As a best practice, consider using the `engine_started` signal handler.
```

Comment:
Similar to the news entry, this shouldn't contrast settings and start_requests with crawler components and engine_started. Both things are true for both things.